Index.html

```diff
@@ -31,7 +31,170 @@ <h3 class="connect">LET’S CONNECT</h3>
         </ul>
       </div>
   </section>
-    
+   <section class="experience">
+    <div>
+      <div class="proj1">
+        <div class="proj-content-space">
+          <h2 class="proj-title">Tonic</h2>
+          <nav>
+            <h4 class="cpny">Canopy</h4>
+            <img src="./images/Counter.png" alt="counter" />
+            <span class="project-lang">Back End Dev</span>
+            <img src="./images/Counter.png" alt="counter" />
+            <span>2015</span>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div> 
+
+      <div class="proj2">
+        <div class="proj-content-space">
+          <h1 class="proj-title">Multi-Post Stroies</h1>
+          <nav class="proj-img2">
+            <h4 class="cpny">Canopy</h4>
+            <img src="./images/Counter.png" alt="counter" />
+            <span class="project-lang">Back End Dev</span>
+            <img src="./images/Counter.png" alt="counter" />
+            <span>2015</span>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div>
```

Comment:
Kindly update the styling of each of your cards to better match the figma design🙏.
Your design:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/37116322/219805833-89be1746-dc63-4db0-b6ec-afad7f6618e7.png">

Figma design:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/37116322/219483113-557ca771-0e25-4c35-8fa0-b8284d450cb3.png">

A few things to note:
- The background color of each card should be white(#fff)
- The contents of the card should be left aligned
- Each card should have a discernable start and end. Currently, your cards flow into each other

**Please try your best to make sure that your design matches the Figma design as much as possible😎**